priv/cabbage/apps/itest/test/features/standard_exits.feature

```diff
@@ -2,6 +2,6 @@ Feature: Standard Exits
   Scenario: Alice starts a Standard Exit
     When Alice deposits "1" ETH to the root chain
     Then Alice should have "1" ETH on the child chain
-    When Alice starts a standard exit on the child chain
+    When Alice completes a standard exit on the child chain
     Then Alice should have "0" ETH on the child chain after finality margin
-    And Alice should have "100" ETH on the blockchain
\ No newline at end of file
+    And Alice should have "100" ETH on the blockchain
```

Comment:
```suggestion
    And Alice should have "1" ETH more on the root chain
```

Reply:
as we chatted - 'more' refers to the very beginning, so Alice has exactly the same amount on the root chain